client/src/pages/Home.js

```diff
@@ -1,7 +1,72 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { fetchCurrentGsi } from "../api/fetchForecast";
+import BackgroundContainer from "../components/BackgroundContainer";
+import SwipeUpFooter from "../components/SwipeUpFooter";
+import styled from "@emotion/styled";
+import ErrorScreen from "../components/ErrorScreen";
+
+const CurrentGsiDisplay = styled.div`
+  font-size: 25vw;
+  color: #dea64c;
+`;
+
+const CityDisplay = styled.div`
+  font-size: 8vw;
+  color: #ffffff;
+  font-style: bold;
+`;
+
+const InfoText = styled.div`
+  font-size: 7vw;
```

Comment:
It's interesting that you use "vw" for font-size. I think they could get very huge on a desktop screen. Make sure to test it on different screen sizes and aspect ratios. For wide-screen displays it might behave differently than for displays in 3:2 aspect ratio, for example.